src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -64,8 +69,37 @@ public void execute_listIsFiltered_showsVideos() {
             listCommand.execute(expectedModel);
         } catch (ParseException e) {
             assertNull(e);
+        } catch (CommandException e) {
+            assertNull(e);
         }
         String expectedString = String.format(ListCommand.MESSAGE_SUCCESS_VIDEOS, moduleCode, lectureName);
         assertCommandSuccess(listCommand, model, expectedString, expectedModel);
     }
+
+    @Test
+    public void execute_listIsFiltered_moduleNotFoundThrowsCommandException() {
+        ModuleCode moduleCode = new ModuleCode(VALID_MODULE_CODE_2103);
+        String input = String.format("list /mod %s", moduleCode);
+        String expectedString = String.format(MESSAGE_MODULE_DOES_NOT_EXIST, moduleCode);
+        try {
+            listCommand = new ListCommandParser().parse(input);
+            assertThrows(CommandException.class, expectedString, () -> listCommand.execute(expectedModel));
+        } catch (ParseException e) {
+            assertNull(e);
+        }
```

Comment:
Might want to simplify this whole chunk by using the `assertCommandFailure` method